src/partition/ReceivedPartition.cpp

```diff
@@ -161,6 +171,14 @@ void ReceivedPartition::compute()
     // A vertex belongs to a specific connected rank if its global vertex ID lies within the ranks min and max.
     std::map<int, std::vector<int>> remoteCommunicationMap;
 
+    // if(utils::MasterSlave::getRank()==0)
+    // {
+    //   for (auto & vtx : _mesh->vertices())
+    //   {
+    //     std::cout<< vtx.getGlobalIndex() << std::endl;
+    //   }
+    // }
```

Comment:
Let's not push this to develop. If you think this is helpful in the long run you could make it a logging output and potentially guard with `#ifndef NDEBUG`.